interfaces/cython/cantera/_onedim.pyx

```diff
@@ -1537,20 +1537,26 @@ cdef class Sim1D:
             return self.sim.fixedTemperatureLocation()
 
     def save(self, filename='soln.yaml', name='solution', description=None,
-             loglevel=None, overwrite=False, compression=0):
+             loglevel=None, *, overwrite=False, compression=0, basis=None):
```

Comment:
Have we released with the old signature? If so, we should note this is a breaking change, specifically changing the two existing args to kwarg-only

Reply:
All new options have been introduced in Cantera 3.0.